projects/kit/directives/skeleton/skeleton.style.less

```diff
@@ -0,0 +1,37 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+@keyframes tuiSkeleton {
+    from {
+        opacity: 0.03;
+    }
+
+    to {
+        opacity: 0.06;
+    }
+}
+
+[tuiSkeleton] {
```

Comment:
Design system also requires smooth switching between skeleton/content.

Could we animate it ?
For example, could we set `.transition(filter)` for host element ?

<img width="801" alt="smooth-animation" src="https://github.com/taiga-family/taiga-ui/assets/35179038/44550bc6-0954-46e4-905e-7e644c90c5d4">

Reply:
There's a problem with setting transition on skeleton directive because there are other transitions applied to the host element. Maybe Angular animations can help us here so it does not interfere with CSS transitions.